packages/catalog-process/test/db.test.ts

```diff
@@ -2132,13 +2178,149 @@ describe("database test", async () => {
             producersIds: [],
             states: [],
             agreementStates: [],
+            attributesIds: [],
           },
           5,
           5
         );
         expect(result.totalCount).toBe(6);
         expect(result.results.length).toBe(1);
       });
+      it("should get the eServices if they exist (parameters: attributesIds)", async () => {
+        const result = await readModelService.getEServices(
+          getMockAuthData(),
+          {
+            eservicesIds: [],
+            producersIds: [],
+            states: [],
+            agreementStates: [],
+            attributesIds: [
+              attributesForDescriptor1and2.certified[0][0].id,
+              attributesForDescriptor3.declared[0][1].id,
+              attributesForDescriptor4.verified[0][1].id,
+            ],
+          },
+          0,
+          50
+        );
+        expect(result.totalCount).toBe(4);
+        expect(result.results).toEqual([
+          eService1,
+          eService2,
+          eService3,
+          eService4,
+        ]);
+      });
+
+      it("should not get the eServices if they don't exist  (parameters: attributesIds)", async () => {
+        const result = await readModelService.getEServices(
+          getMockAuthData(),
+          {
+            eservicesIds: [],
+            producersIds: [],
+            states: [],
+            agreementStates: [],
+            attributesIds: [generateId()],
+          },
+          0,
+          50
+        );
+        expect(result.totalCount).toBe(0);
+        expect(result.results).toEqual([]);
+      });
+
+      it("should get the eServices if they exist (parameters: attributesIds, name)", async () => {
```

Comment:
The rationale behind _"which combination of parameters to test"_ was the following:
- test each filter singularly,
- test the combinations that are used in frontend: @Carminepo2 helped me identify `{name, agreementStates, states}` and `{states, name, producerIds}`. Not sure where to add the new filter for `attributeIds`, but if you want we could take a look so that we don't include non-possible combinations.

P.S. @galales the `GET eServices` endpoint is becoming too flexible and a bit complicate to maintain. I am thinking of proposing to split it, so that there will be a couple of endpoints that require only the combinations of parameters that are actually in use. We could start this conversation when you have some time.

Reply:
sure! could you collect all the use cases in the as-is version?

Reply:
I will do! In the meantime I've just created IMN-242 in the board.

Reply:
Sorry for the late response. 
As far as I can see, there is no use of the filter `attributesIds` in the front-end

Reply:
✔️  maybe we can leave the tests as they are, so that they test this filter and its usage together with some other filters, then refine them when we know the exact use cases, @taglioni-r what do you think?
Maybe @beetlecrunch or @galales have more context on this filter?

Reply:
Sure Eric, let's keep your tests as you wrote.